### PR TITLE
Test Samples Landing page added

### PR DESCRIPTION
**Pull request by Dheeraj-DJP**
1. Basic structure for Test Samples Page has been added.
2. Created a page for Completed Test Page also.
3. Dummy json used to create table and user information.
4. Used bootstrap like grid layout instead of {Row,Col} layout usually
   used in React
5. Checkbox not added in Table content

### CODE REVIEW

File: src/components/NavBarLanding/NavBarLanding.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@import '../../constants/colors.scss';
+
+.navbar {
+    background-color: $background;
```
Comment (ankitamasand) [inline src/components/NavBarLanding/NavBarLanding.scss:4]: Let's name color variables as `$color-background`. In that way, it is easy to identify that this is variable is color. Going forward, we would have a few more variables coming in!

File: src/components/NavBarLanding/VectorNotif.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="21" viewBox="0 0 20 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ankitamasand) [inline src/components/NavBarLanding/VectorNotif.svg:1]: Let's put svgs in `assets` folder

File: src/services/route/routes2.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import Wrapper from '../../bootstrap/Wrapper/Wrapper';
+import App from '../../App';
+import Login from '../../views/LoginSetup/Login';
+import SignUp from '../../views/LoginSetup/SignUp';
+import RequestSent from '../../views/LoginSetup/RequestSent';
+import TestSamples from '../../views/LandingPage/TestSamples';
+import CompletedTests from '../../views/LandingPage/CompletedTests';
+
+const routes = [{
```
Comment (ankitamasand) [inline src/services/route/routes2.js:9]: why create `routes2.js`? We can add these landing page routes in `routes.js` file.

Comment (Dheeraj-DJP) [reply src/services/route/routes2.js:9]: I forgot to delete that. routes2.js is not used.

File: src/components/NavBarLanding/VectorNotif.svg (hunk 2 above)
Comment (Dheeraj-DJP) [reply src/components/NavBarLanding/VectorNotif.svg:1]: The 'assests' folder should be created in src folder?

Comment (ankitamasand) [reply src/components/NavBarLanding/VectorNotif.svg:1]: yes, let's create an `assets` folder in `src` and we can put all the SVGs there. Also, let's write a helper function to import an SVG by name. For example, you can have:
```javascript
getSvg('notify')
```
In this case, we only have to import the `getSvg` helper method and not multiple SVG files in our views.

File: src/views/LoginSetup/Login/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class Login extends Component {
```
Comment (ankitamasand) [inline src/views/LoginSetup/Login/index.js:3]: why are these files added in this PR?

**Comment by Dheeraj-DJP** [comment]
Changes to be made in the files added to the draft PR. 
Opening a new PR under a different branch. 
Closing the draft PR.
